### PR TITLE
Revamped Plan & Pricing Page

### PR DESCRIPTION
**Pull request by OAGr**
image

### CODE REVIEW

File: src/components/plans/index/questions.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    header: "What is a model?",
+    body: <p>A model is a single spreadsheet made for one main computation.  One example is <a href='/scratchpad'>here</a>.</p>,
+  },
+  {
+    header: "Who can access a private models?",
+    body: <p>Individual private models can only be accessed by that individual.  Organizational private models can be seen and edited by all members of that organization.</p>,
+  },
+  {
+    header: "Is there a minimum contract?",
+    body: <p>All plans are monthly and can be canceled or changed at any time.</p>
```
Comment (mmcdermott) [inline src/components/plans/index/questions.js]: What about minimum # of Users for organization plan?

File: src/components/plans/index/questions.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  {
+    header: "What happens after I end a plan?",
```
Comment (mmcdermott) [inline src/components/plans/index/questions.js]: Maybe
'What happens if I unsubscribe?'
would be a better header?

File: src/components/plans/index/questions.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    body: <p>Your private models will be disabled until you begin a plan again.  This means they will not be viewable or editable.</p>
+  },
+  {
+    header: "What payment methods do you accept?",
+    body: <p>We accept all Visa, American Express, and Mastercard credit cards.</p>
+  },
+  {
+    header: "Do you offer educational or nonprofit discounts?",
+    body: <p>Yes.  Please contact us for more information.</p>
```
Comment (mmcdermott) [inline src/components/plans/index/questions.js]: contact us should be a link to emailing us or something.

File: src/routes/layouts/login/index.js [hunk 4]
```diff
@@ -48,7 +48,7 @@ export default class Profile extends Component {
 
     let listElements = [
       {ionicIcon: 'md-person', header: 'account', onMouseDown: this._openModal.bind(this)},
-      {icon: 'rocket', header: 'upgrade', onMouseDown: () => {navigationActions.navigate('/pricing')}},
+      {icon: 'rocket', header: 'upgrade', onMouseDown: () => {navigationActions.navigate('/subscribe/lite')}},
```
Comment (mmcdermott) [inline src/routes/layouts/login/index.js]: Don't you want this to be dependent on the user's plan?

File: src/components/plans/index/PlanIndex.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React, {Component, PropTypes} from 'react'
```
Comment (mmcdermott) [inline src/components/plans/index/PlanIndex.js]: Maybe there should be some callout at the top of the page if they are logged out that says 'log in to select a plan' just so viewers know they have to be signed in.

File: src/components/plans/index/questions.js (hunk 3 above)
Comment (OAGr) [inline src/components/plans/index/questions.js]: It's not shown at all if they don't have a plan.

File: src/routes/layouts/login/index.js (hunk 4 above)
Comment (OAGr) [inline src/routes/layouts/login/index.js]: It's not shown at all if they don't have a plan.

File: src/components/plans/index/PlanIndex.js (hunk 5 above)
Comment (OAGr) [inline src/components/plans/index/PlanIndex.js]: Let's worry about this later.  One step at a time.

**Pull request merged**